WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingsViewController.swift

```diff
@@ -116,15 +116,13 @@ class NotificationSettingsViewController: UIViewController {
 
         activityIndicatorView.startAnimating()
 
-        if AppConfiguration.showsFollowedSitesSettings {
```

Comment:
This is the only flag I decided to just remove because it's in the incorrect place – WordPress has no notification settings to begin with. If we need more control for Reader, which we can't tell now, we'll add more flags.